Add more 3rd party plugins to docs (#426)
jellyfin/jellyfin-docs: general/server/plugins/index.md

* [GitHub](https://github.com/TheJF/antennas)

#### Merge Versions

Automatically group every repeated movie.

**Links:**

* [GitHub](https://github.com/danieladov/jellyfin-plugin-mergeversions)

#### Intros

Download flashy intros from prerolls.video for your movies.

**Links:**

* [GitHub](https://github.com/dkanada/jellyfin-plugin-intros)

> #### Youtube Metadata

Reviewer: ```suggestion
#### YouTube Metadata
```

Reviewer: I think you missed this one :smile:

Author: Oops. Im still figuring out how pull requests work.

Reviewer: No worries, thanks for the fix!

> Download metadata about Youtube videos using a Youtube API key.

Reviewer: ```suggestion
Downloads metadata of YouTube videos with a YouTube API key.
```

**Links:**

* [GitHub](https://github.com/ankenyr/jellyfin-youtube-metadata-plugin)

#### Last.FM

Enables audio scrobbling to Last.FM as well as a metadata fetcher source. 

**Links:**

* [GitHub](https://github.com/jesseward/jellyfin-plugin-lastfm)

#### VGMdb

Adds support for VGMdb to music libraries. Can provide both images and metadata for artists and albums.

**Links:**

> * [GitHub](https://github.com/nielsvanvelzen/jellyfin-plugin-vgmdb)

Reviewer 2: Hey I know this one!
